### DISCUSSION TITLE
Portuguese translation problem

### DISCUSSION THREAD

**Discussion by thiagoaf19** [dbeaver/dbeaver#19397] [Q&A]
### Description

I have a problem translating into Portuguese, can anyone help me?
Grateful
![01](https://user-images.githubusercontent.com/72985546/223755855-de5af959-ed8f-49ba-901e-02a51cde6823.JPG)

### DBeaver Version

23.0.0.202303040621

### Operating System

Windows 10

**Comment by E1izabeth** [comment]
Hi, @thiagoaf19!
You can help us make DBeaver better :) Here is useful wiki of how to add new translation https://github.com/dbeaver/dbeaver/wiki/Localization
If you need any help, you can ask here.

**Comment by thiagoaf19** [comment]
didn't help me, I need a correct translation.

**Comment by E1izabeth** [comment]
I don't know Portuguese, but maybe someone from our community might help you

**Comment by thiagoaf19** [comment]
> Não sei português, mas talvez alguém da nossa comunidade possa te ajudar

yes, I'm waiting.

**Comment by E1izabeth** [comment]
Probably, you can repeat your question in Discussions. Maybe someone will find it there.
